src/extensions/windows/Microsoft.DotNet.UpgradeAssistant.Extensions.Windows/WinformsDefaultFontUpdater.cs

```diff
@@ -7,23 +7,29 @@
 using System.IO;
 using System.Linq;
 using System.Text;
+using System.Text.Json;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Xml.XPath;
 using Microsoft.Extensions.Logging;
 
 namespace Microsoft.DotNet.UpgradeAssistant.Extensions.Windows
 {
     [ApplicableComponents(ProjectComponents.WinForms)]
     public class WinformsDefaultFontUpdater : IUpdater<IProject>
     {
+        private const int BufferSize = 65536;
+        private const string HighDPIConfiguration = "/configuration/System.Windows.Forms.ApplicationConfigurationSection";
+
         private readonly WindowsUtilities _utilities = new();
         private readonly ILogger<WinformsDefaultFontUpdater> _logger;
+        private readonly string _winformsSourceUpdateInfo = "Default font in Windows Forms has been changed from Microsoft Sans Serif to Seg Segoe UI, in order to change the default font use the API - Application.SetDefaultFont(Font font). For more details see here - https://devblogs.microsoft.com/dotnet/whats-new-in-windows-forms-in-net-6-0-preview-5/#application-wide-default-font. \nIts recommended to set HighDpiMode to be SystemAware for better results in Main() - Application.SetHighDpiMode(HighDpiMode.SystemAware)";
```

Comment:
This should be a resource string

Reply:
Also, should it be two separate resource strings since these are two separate messages we want to give users?